### PR TITLE
New package for FStar 0.9.2.0

### PR DESCRIPTION
**Pull request by msprotz**

### CODE REVIEW

**Comment by beurdouche** [comment]
It looks that the CI failure for `continuous-integration/travis-ci/pr` is because there are no GNU versions of `install` installed on the system.
Note that the build will succeed if `ginstall` is present or if `install` is the GNU version instead of the BSD version. This can be installed on the CI machine through any package manager.

**Comment by yallop** [comment]
Could you please add 

`bug-reports: "https://github.com/FStarLang/FStar/issues"`

to the `opam` file and, if possible, split up the `build` field into separate `build` and `install` components?

**Comment by dsheets** [comment]
To install GNU `install`, I believe the appropriate depext is

```
depexts: [
  [["osx" "homebrew"] ["coreutils"]]
]
```

**Comment by msprotz** [comment]
New commit with the suggested fixes.

**Comment by msprotz** [comment]
\o/ thanks @dsheets and @yallop for the tips & tricks

File: packages/fstar/fstar.0.9.2.0/url [hunk 1]
```diff
@@ -0,0 +1,2 @@
+archive: "https://github.com/FStarLang/FStar/archive/0.9.2.0.tar.gz"
+checksum: "813242eaababde5c350e3205d53aa837"
```
Comment (beurdouche) [inline packages/fstar/fstar.0.9.2.0/url]: @msprotz, this is not the correct package right ?

**Pull request merged**